app.py

```diff
@@ -43,11 +43,38 @@ def experience():
         
         data['experience'].append(new_experience)
         save_data('data/data.json', data)
-        
         return jsonify({'id': new_id}), 201
     
+    if request.method == 'DELETE':
+        index = request.args.get("index")
+        # check index is valid number
+        if index is not None:
+            if not index.isnumeric():
+                return jsonify({"error": "Invalid Index"}), 400
+            
+        if 0 < int(index) <= len(data["experience"]):
+            # ids in data.json are 1 indexed
+            data["experience"].pop(int(index)-1)
+            save_data('data/data.json', data)
+            return jsonify({"message": "Successfully deleted"}), 200
+
+        return jsonify({"error": 'Invalid Index'}), 400
+
     return jsonify({'error': 'Method not allowed'}), 405
 
+
+@app.route('/resume/experience/<int:index>', methods = ['GET'])
```

Comment:
iirc we're not using index as a url param, but similar to what you have in the DELETE endpoint